src/renderer/modules/webpack/helpers.ts

```diff
@@ -246,7 +248,7 @@ export function getByValue<T>(
   return getModule<T>(filters.byValue(match), options);
 }
 
-export function getByStoreName<T extends Store>(name: string): T | undefined {
+export function getByStoreName<T>(name: string): (T & Store) | undefined {
   const stores = Flux.Store.getAll();
-  return stores.find((store) => store.getName() === name) as T | undefined;
+  return stores.find((store) => store.getName() === name) as (T & Store) | undefined;
 }
```

Comment:
Seems like an unrelated change

Reply:
yeah was frustrated on that so fixed it kinda. was in webpack too so thought would be fine in this update. this will make it so generic wouldn't have to extend store

Reply:
That should not be in this PR though.